### ISSUE TITLE
WCAG 1.4.3: Elements must have sufficient color contrast (.sidebar-item-title)

### ISSUE THREAD

**Issue by brianteeman** [joomla-projects/j4adminui#55]
#### Issue
`Elements must have sufficient color contrast` ([color-contrast](https://dequeuniversity.com/rules/axe/3.3/color-contrast?application=msftAI))
#### Target application
[Control Panel - kh - Administration](http://localhost/kh/administrator/index.php)
#### Element path
.active > .sidebar-item-title
#### Snippet
`<span class="sidebar-item-title">Dashboard</span>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 3.21 (foreground color: #0184ff, background color: #e5f2ff, font size: 12.0pt (16.002px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 78.0.3904.97
